client/src/components/SpendingForm.js

```diff
@@ -50,151 +44,134 @@ const Title = styled.p`
 const Container1 = styled.div`
   display: flex;
   justify-content: space-between;
-`;
-
-const Container2 = styled.div`
-  display: flex;
-  justify-content: center;
-  width: 100%;
-`;
-
-const Container3 = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  margin-bottom: 40px;
-`;
-
-const Headings = styled.p`
-  all: unset;
-  margin-top: 10px;
-  font-size: 14px;
-  margin-right: auto;
-  margin-left: 5px;
-`;
-
-const CategorysContainer = styled.div`
-  display: flex;
-  align-items: space-between;
-  margin-top: 5px;
   width: 100%;
 `;
 
-const CategoryTitle = styled.p`
-  all: unset;
-  font-size: 7pt;
-  color: ${light.colors.fontsecondary};
-  text-align: center;
-  margin-top: 10px;
-`;
-
 const CategoryContainer = styled.div`
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: space-between;
   width: 100%;
-  margin-left: 0;
-  margin-bottom: 10px;
+  margin-top: -20px;
 `;
 
-const styleFirstInput = {
-  marginLeft: 0
+const styleFormItem = {
+  flex: '0 0 48%'
 };
 
-const styleSecondInput = {
-  marginRight: 0
-};
+const categories = [
```

Comment:
I'm really not sure but maybe you can make something like
```
const categories = [

fill: light.colors.action

{

icon : "food",
value: "food"

},
{

icon: "transport",
value: "transport"

}
...
```
But thats the matter i'm struggling with at the moment (Destructuring) so take this proposal with caution  🙈